### DISCUSSION TITLE
如何将后面的章节、目录等变为英文

### DISCUSSION THREAD

**Discussion by IdealSheaf** [tuna/thuthesis#957] [Q&A]
我想用英语写论文，需要把后面目录、章节等内容全部改为英文。我尝试在documentclass里加上language=english，但是这样会导致封面出现两个问题：1、多出现了一个英文标题，显示为Thesis Title: XXX，但是中文的标题前就没有“论文题目”四个字 2、多了这两行英文标题导致下面的作者、导师等内容全部跳到下一页了。希望可以问题可以得到解答，谢谢！

**Comment by Harry-Chen** [comment]
你使用的模版是什么版本？是否太老了？

**Comment by zepinglee** [comment]
本科生还是研究生？给个复现的代码示例？
